coral/src/app/layout/Layout.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box, Grid } from "@aivenio/design-system";
+import MainNavigation from "src/app/layout/MainNavigation";
+import Header from "src/app/layout/Header";
+import SkipLink from "src/app/layout/SkipLink";
+import { useRef } from "react";
+
+function Layout({ children }: { children: JSX.Element }) {
+  const ref = useRef<HTMLDivElement>(null);
+  return (
+    <>
+      <SkipLink mainContent={ref} />
+      <Grid
+        colGap={"l1"}
+        minHeight={"screen"}
+        style={{
+          gridTemplateColumns: "240px 1fr",
+          gridTemplateRows: "auto 1fr",
+        }}
+      >
+        <Header />
```

Comment:
Currently we define the following in `Header.tsx`:
```
    <GridItem
      htmlTag={"header"}
      colStart={"1"}
      colEnd={"12"}
      height={"l5"}
      backgroundColor={"primary-80"}
      paddingX={"l2"}
    >
```

I think we could consider the `<Grid />` and `<GridItem />` in a same way as `<ul>` and `<li>`. I think the `<Layout />` component should be responsible of _layouting_ (🤦 ) and the `<Header />` should concentrate on the content. Now I feel that there is a coupling between `<Layout />` and `<Header />` as layout defines a slot and header tells how it should fit there. 

```
<Grid>
<GridItem><Header />....
```